oecophylla/taxonomy/parser.py

```diff
@@ -1,7 +1,26 @@
+import sys
 import pandas as pd
 import biom
 from biom.util import biom_open
-import sys
+
+
+def combine_profiles(profiles):
+    """Combines profiles for several samples into one table.
+
+    Parameters
```

Comment:
can you say something about the assumed file format of the per sample files?

Reply:
Yes I did. In line #14.

Reply:
I did not look closely enough. Sorry. But you might also describe it a bit more elaborate.

Reply:
No problem! `feature<tab>count` is all one needs to know about the format, I think. There is no other ingredients.

Reply:
oh, I meant in the code :-)